mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -14,24 +14,33 @@
 import xarray as xr
 import datetime
 from netCDF4 import Dataset as netcdf_dataset
-import os.path
 
-from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
+from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, \
+    remove_repeated_time_index
 from ..shared.plot.plotting import plot_global_comparison
 from ..shared.interpolation.interpolate import interp_fields, init_tree
 from ..shared.constants import constants
 
 from ..shared.io import StreamsFile
 
-def ocn_modelvsobs(config, field):
+
+def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
 
     """
     Plots a comparison of ACME/MPAS output to SST or MLD observations
 
+    If present, streamMap is a dictionary of MPAS-O stream names that map to
+    their mpas_analysis counterparts.
+
+    If present, variableMap is a dictionary of MPAS-O variable names that map
+    to their mpas_analysis counterparts.
+
     Authors: Luke Van Roekel, Milena Veneziani, Xylar Asay-Davis
-    Modified: 10/27/2016
+    Modified: 12/07/2016
     """
 
+    field = field.lower()
```

Comment:
Again, a bit of mission creep.  I didn't decide that this was the way, I just moved this to the top because every time `field` was accessed it was as `field.lower()`.  I agree, though, that it should just always be called with a lowercase variable name if that's what we want.

I'll take this out, though, because I agree that it doesn't really make sense to me.

@vanroekel or @milenaveneziani, can you explain why it was important that the field name be all lowercase?